generators/server/templates/src/main/java/package/client/_OAuth2InterceptedFeignConfiguration.java

```diff
@@ -41,6 +38,6 @@ public OAuth2InterceptedFeignConfiguration(LoadBalancedResourceDetails loadBalan
 
     @Bean(name = "oauth2RequestInterceptor")
     public RequestInterceptor getOAuth2RequestInterceptor() throws IOException {
-        return new OAuth2FeignRequestInterceptor(new DefaultOAuth2ClientContext(), loadBalancedResourceDetails);
+        return new FeignClientInterceptor(new DefaultOAuth2ClientContext(), loadBalancedResourceDetails);
```

Comment:
This change looks wrong, shouldn't that use the OAuth2FeignRequestInterceptor from spring like before?

Reply:
sorry, it was old code.